doc/interactivity/CMakeLists.txt

```diff
@@ -37,6 +37,12 @@ install(
   LIBRARY
   DESTINATION
     ${CATKIN_PACKAGE_LIB_DESTINATION}
+  ARCHIVE
+  DESTINATION
+    ${CATKIN_PACKAGE_LIB_DESTINATION}
+  RUNTIME
+  DESTINATION
+    ${CATKIN_PACKAGE_LIB_DESTINATION}
```

Comment:
According to https://github.com/ros-planning/moveit/pull/1575#issuecomment-514296131, this should be `${CATKIN_GLOBAL_BIN_DESTINATION}`, shouldn't it?

Reply:
Good catch! Fixed it in the latest commit.